guides/installation/requirements.md

```diff
@@ -63,13 +63,18 @@ You can use these commands to check your actual environment:
 
 The `media:delete-unused` command requires at least MariaDB 10.9 or MySQL 8.
 
-For optimal MySQL performance, it is advisable to have minimum of 32 MB.
+For optimal MySQL performance, it is advisable to have a minimum of 32 MB.
 
 ### JavaScript
 
 * Node.js 18.0.0 or higher
 * NPM 8.0.0 or higher
 
+## Redis
+
+* Version: 6.2 or higher
```

Comment:
@shyim Is this a compatible or recommended version? We have categorized all versions as Recommended, Compatible, or Problematic.

Reply:
done